content-test/components/SiteIcon.test.js

```diff
@@ -104,3 +114,35 @@ describe("SiteIcon", () => {
     });
   });
 });
+
+describe("SiteIcon loaded", () => {
+  let node;
+  let instance;
+
+  function setup(customProps = {}, customSiteProps = {}) {
+    const site = Object.assign({}, fakeProps.site, customSiteProps);
+    const props = Object.assign({}, fakeProps, customProps, {site});
+    node = document.createElement("div");
+    instance = ReactDOM.render(<SiteIcon {...props} />, node);
+  }
+
+  afterEach(() => {
+    ReactDOM.unmountComponentAtNode(node);
+  });
+
+  it("should use fallback on small 1x1 favicon", done => {
+    setup({}, {favicon_url: "data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAYAAAAfFcSJAAAAEElEQVR42gEFAPr%2FAP%2F%2F%2F%2F8J%2BwP9vTv7fQAAAABJRU5ErkJggg%3D%3D"});
```

Comment:
It's a separate setup as the instance is actually rendered to the document triggering DOM events.